src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -99,7 +99,7 @@ abstract class Generator
 		private INamedTypeSymbol _iOSBaseSymbol;
 		private INamedTypeSymbol _androidBaseSymbol;
 		private INamedTypeSymbol _macOSBaseSymbol;
-		private Compilation _referenceCompilation;
+		private static Compilation s_referenceCompilation;
```

Comment:
Making this static to load it only once. It doesn't depend on any state (i.e., `LoadUWPReferenceProject(@"..\..\..\Uno.UWPSyncGenerator.Reference\references.txt")`)

Reply:
Not only that this will load the project once, but also will re-use things that Roslyn has already cached.

Reply:
I'm not seeing a significant improvement by that change in CI time. :/

Reply:
That's fine, it may have an impact on memory.